lib/cli.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+
+/**
+ * This file process should arguments the terminal
+ * @return {String}
+ *   The Readed String.
+ */
+function process() {
+  var command = 'read';
+  console.log('Fake reading terminal arguments');
+  return command;
+}
+
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/9916b12f4cf7eef2d0ef8e80a0791204126cba3e